src/external/rawspeed/rawspeed-identify.cpp

```diff
@@ -119,7 +119,11 @@ int main(int argc, const char* argv[])
 
     fprintf(stderr, "Loading file: \"%s\"\n", argv[1]);
 
+    #if defined (_WIN32)
```

Comment:
Illegal how? Against standard - yes. Does any compiler with default settings say anything about definition of such a define? No, only (?) clang has non-default warning. So user manipulation with these defines should be classified as broken build environment, surely nothing we can do anything about.
And yes, if this is specific to Microsoft compilers, then we should not worry.